examples/solomo/client/routes.js

```diff
@@ -1,21 +1,55 @@
-Router.map(function() {
-  this.route('map', {
-    path: '/',
-    data: function () {
-      Session.set("currentPhoto", null);
-    }
-  });
-  this.route("mapWithPhoto", {
-    template: "map",
-    path: 'map/:_id',
-    data: function () {
-      Session.set("currentPhoto", this.params._id);
-    }
-  });
-  this.route('camera-page');
-  this.route("list");
-});
+import { FlowRouter } from 'meteor/ostrio:flow-router-extra';
+import { Session } from 'meteor/session'
 
-Router.configure({
-  layoutTemplate: "layout"
-});
\ No newline at end of file
+FlowRouter.route('/', {
+  name: 'index',
+  action: function () {
+    this.render('layout', 'map')
+  },
+  data: function () {
+    Session.set("currentPhoto", null);
+  }
+})
+FlowRouter.route('/map/:_id', {
+  name: 'mapWithPhoto',
+  action: function () {
+    this.render('layout', 'map')
+  },
+  data: function (params) {
+    Session.set("currentPhoto", params._id);
+  }
+})
+FlowRouter.route('/camera-page', {
+  name: 'camera-page',
+  action: function () {
+    this.render('layout', 'camera-page')
+  }
+})
+FlowRouter.route('/list', {
+  name: 'list',
+  action: function () {
+    this.render('layout', 'list')
+  }
+})
+
+// Router.map(function() {
+//   this.route('map', {
+//     path: '/',
+//     data: function () {
+//       Session.set("currentPhoto", null);
+//     }
+//   });
+//   this.route("mapWithPhoto", {
+//     template: "map",
+//     path: 'map/:_id',
+//     data: function () {
+//       Session.set("currentPhoto", this.params._id);
+//     }
+//   });
+//   this.route('camera-page');
+//   this.route("list");
+// });
+//
+// Router.configure({
+//   layoutTemplate: "layout"
+// });
```

Comment:
why do we keep these comments?

Reply:
I would like to keep it commented for at least one more iteration just in case I have missed something in the migration to flow-router.